paintingprocessmonitoring/src/main/resources/application.yml

```
@@ -73,12 +270,33 @@ spring:
+
+# JWT 설정
+jwt:
+  secret: ${JWT_SECRET:smartFactorySecretKeyForProductionEnvironmentUseOnly2024}
+  expiration: 86400000  # 24시간
 
+# 애플리케이션 설정
+app:
+  security:
+    jwt:
+      secret-key: ${JWT_SECRET:smartFactorySecretKeyForProductionEnvironmentUseOnly2024}
+      expiration-time: 86400000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)